include/roboteam_ai/control/BBTrajectories/BBTrajectory1D.h

```diff
@@ -0,0 +1,124 @@
+//
+// Created by rolf on 15-01-20.
+//
+
+#ifndef RTT_BBTRAJECTORY1D_H
+#define RTT_BBTRAJECTORY1D_H
+
+/**
+ * @author rolfvdhulst
+ * @date 15 January 2020
+ * @tparam num : float or double
+ */
+template<class num>
+struct BBTrajectoryPart {
+  num tEnd;
+  num acc;
+  num startVel;
+  num startPos;
+};
+
+/**
+ * @author rolfvdhulst
+ * @date 15 January 2020
+ * @tparam num : float or double
+ */
+template<class num>
+struct PosVelAcc {
+  explicit PosVelAcc(num pos, num vel, num acc)
+          :
+          pos{pos},
+          vel{vel},
+          acc{acc} { };
+
+  num pos;
+  num vel;
+  num acc;
+};
+
+/**
+ * @author rolfvdhulst
+ * @date 15 January 2020
+ * @tparam num : float or double
+ */
+template<class num>
```

Comment:
```suggestion
template<typename Num>
```
Capitals are the convention for templates